### ISSUE TITLE
[AzureTools][Aspire] The displayed information by hovering over “Where do I find the token?” is not adaptive when reducing the size of the browser page

### ISSUE THREAD

**Issue by v-mengwe** [dotnet/aspire#4171]
**Clean machine:** Win11 x64 23h2 ENU
**VS Version:** VS 17.11.0 Preview 2.0 [34913.145.main]
**Aspire Version:** 8.1.0-preview.1.24259.4
Apply NuGet Feeds

**REPRO STEPS:**
1. Make sure there is no localhost in your browser.
2. Open Terminal, run the following commands to create and run the project using https:
```
dotnet new aspire -o aspire
cd aspire/aspire.AppHost
dotnet run
```
3. Reduce the size of your browser page by dragging and dropping with your mouse -> Click the link without token in the command prompt, move mouse to 'Where do I find the token?'.

**Expect:**
The displayed information is adaptive.

**Actual:**
The displayed information is not adaptive. (Only part of the information is displayed, the browser must be enlarged to display the full message.)
![Actu](https://github.com/dotnet/aspire/assets/62053665/09017def-0977-43f0-82ad-62adc05d6e96)

**More info:**
This issue also **reproduces** on VS 17.11.0 Preview 2.0 [34913.145.main] with Aspire 8.0.0.

**Comment by leslierichardson95** [comment]
If this is a quick fix, can we try to complete this issue in 8.1?  Otherwise, let's punt it to 8.2 or backlog depending on the severity.

**Comment by tlmii** [comment]
Current implementation uses a ToolTip, pinned to Bottom. Probably need to switch it to an Anchored Region instead, which would provide the more dynamic placement we're looking for here. Shouldn't be too hard, but more than just changing the position property like we discussed during triage. I'll move to 8.2 but remove myself since I likely won't be around for that work.
